spring-cloud-skipper-server-core/src/main/java/org/springframework/cloud/skipper/server/db/migration/SkipperFlywayConfigurationCustomizer.java

```diff
@@ -69,11 +84,25 @@ else if (databaseDriver == DatabaseDriver.DB2) {
 	private DatabaseDriver getDatabaseDriver(DataSource dataSource) {
 		// copied from boot's flyway auto-config to get matching db vendor id
 		try {
-			String url = JdbcUtils.extractDatabaseMetaData(dataSource, "getURL");
+			String url = JdbcUtils.extractDatabaseMetaData(dataSource, DatabaseMetaData::getURL);
 			return DatabaseDriver.fromJdbcUrl(url);
 		}
 		catch (MetaDataAccessException ex) {
 			throw new IllegalStateException(ex);
 		}
 	}
+
+	private boolean vendorIsMySql(DataSource dataSource) {
+		try {
+			return Optional.ofNullable(ReflectionUtils.findMethod(dataSource.getClass(), "getJdbcUrl"))
```

Comment:
This impl is inconsistent w/ the same one in SCDF (I will make consistent before merging either).

I am actually going to move this logic into a DatabaseMetadata class that is in common flyway and both of these can use the same code (like I did for the flyway EPP).